components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java

```diff
@@ -78,6 +78,27 @@ public Object info(
             return result;
         });
     }
+    
+    @ResponseBody
+    @GetMapping(
+        value = "/{id}",
+        produces = {
+            ActuatorMediaTypes.APPLICATION_ACTUATOR_V1_JSON_VALUE,
+            MediaType.APPLICATION_JSON_VALUE
+        }
+    )
+    public Object routeInfo(
+            @PathVariable String id) {
```

Comment:
maybe this method should be just named ``info`` ? and the previous one for detailed report ``detail``